public/less/variables.less

```diff
@@ -11,3 +11,12 @@
 @brand-info: #5bc0de;
 @brand-warning: #f0ad4e;
 @brand-danger: #d9534f;
+
+//Cross nav banner
+@cross-nav-bg: #e2dfdf; 
+@cross-nav-hover-bg: #f4f4f4;
+@cross-nav-border: #dddddd;
+@cross-nav-list-text: #333333;
+
+//Main menu
+@main-menu-hover: #eb6864;
```

Comment:
Sorry, being a bit of a stickler here, but since you're making changes anyway, might as well fix this little thing.  Our style guide (as encapsulated in `.editorconfig`) dictates `insert_final_newline` which means the file should end with a newline, which currently this file doesn't.